src/dimension_slice.c

```diff
@@ -470,12 +481,20 @@ dimension_slice_fill(TupleInfo *ti, void *data)
 
 /*
  * Scan for an existing slice that exactly matches the given slice's dimension
- * and range. If a match is found, the given slice is updated with slice ID.
+ * and range. If a match is found, the given slice is updated with slice ID
+ * and the tuple is locked.
+ *
+ * Returns the number of tuples found (and locked).
```

Comment:
I think this should return `bool` because there should never be more than one slice found so the only viable return values are true or false (found or not found). The current comment is misleading because it gives the impression that more than one tuple could be found.